InventoryManager/Model/Product.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace InventoryManager.Model
+{
+    /// <summary>
+    /// Stores Product information
+    /// </summary>
+    public class Product
+    {
+        /// <summary>
+        /// Product Id
+        /// </summary>
+        public int Id { get; set; }
```

Comment:
```suggestion
        public int Id { get; set; }
        
```
gap between each declaration